lib/presentation/home_screen/widgets/drawer_widget.dart

```diff
@@ -116,14 +131,14 @@ class SettingsScreen extends StatelessWidget {
   }
 }
 
-class FAQSScreen extends StatelessWidget {
-  const FAQSScreen({super.key});
+// class FAQSScreen extends StatelessWidget {
```

Comment:
I think this should be deleted - we are not going to keep the FAQScreen in the drawer widget.

Reply:
Okay already deleted on the new commit ,

thanks for the PR Advice i will do that in the future,